cmd/livepeer/starter/starter.go

```diff
@@ -1311,6 +1308,14 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 					}
 				}
 
+				// For now, we assume that the version served by the orchestrator is the lowest from all remote workers
+				runnerVersion := worker.LowestVersion(n.AIWorker.Version(), config.Pipeline, config.ModelID)
```

Comment:
I agree with this because things will likely continue changing quickly so it’s important to use worst case scenario using lowest version number.

Reply:
🤝